vendor/assets/javascripts/browser-update.js.erb

```diff
@@ -21,10 +21,14 @@ var $buo = function() {
   document.getElementsByTagName('body')[0].classList.add("crawler");
   var mainElement = document.getElementById("main");
   var noscriptElements = document.getElementsByTagName("noscript");
-  // noscriptElements[0].innerHTML contains encoded HTML
-  var innerHTML = noscriptElements[0].childNodes[0].nodeValue;
-  mainElement.innerHTML = innerHTML;
-
+  // find the element with the "data-path" attribute set
+  for (var i = 0; i < noscriptElements.length; ++i) {
+    if (noscriptElements[i].dataset["path"]) {
```

Comment:
Should we exit the loop once we've found what we were looking for or are there more than one possible match?

Reply:
There is one match and in general there are 1-2 `noscript` tags.